lib/liftoff/scheme_builder.rb

```diff
@@ -0,0 +1,50 @@
+module Liftoff
+  class SchemeBuilder
+
+    def initialize(xcode_project, config)
+      @xcode_project = xcode_project
+      @config = config
+    end
+
+    def create_schemes
+      @xcode_project.generate_scheme # generates the default scheme
```

Comment:
I don't think it's a good idea, because the same method is called to generate custom schemes.